### DISCUSSION TITLE
Carousel 走马灯 API文档缺失 & dots 用法请教

### DISCUSSION THREAD

**Discussion by TinsFox** [ant-design/ant-design#33645] [Q&A]
1. `是否显示面板指示点，如果为 object 则同时可以指定 dotsClass 或者`

**或者啥呀**

![image](https://user-images.githubusercontent.com/33956589/148732226-b0f1f908-e22c-4748-a3f5-e64ef1171dea.png)

2. dots 不能传入一个 className吗？
![image](https://user-images.githubusercontent.com/33956589/148732451-3599bdcb-4553-45e9-b157-c99c886c604a.png)
3. dots 不能传入一个 css object 吗
![image](https://user-images.githubusercontent.com/33956589/148732711-f76125df-5c98-4f2e-9d5f-a4915883e27a.png)

**Comment by dxsun97** [comment]
```jsx
<Carousel dots={{ className: "red" }}>
```
这样就可以指定指示点外层容器的class
